src/domain/models.py

```diff
@@ -9,3 +11,29 @@ class HumanTextInput:
 @dataclass
 class BotTextOutput:
     message: str
+
+
+@dataclass
+class UserQuestion:
```

Comment:
Maybe we can use Request / Response instead Question / Answer model?   
Cause as I can see ( for example in this [user story,](https://github.com/trollknurr/hard-chill-response/issues/10) user can request summary or another fancy solution, not only qustion?

Reply:
My guess is that it should be separate port, for usage with custom prompts
Also i see in future moving toward Agent, and there will be port for any user phrase.
Maybe at that time we will remove this port and leave it only as a service?